apache/trafficcontrol
# Remove dependencies on internal-only Sun packages

## What does this PR (Pull Request) do?
Replaces all references to the internal-only Sun Java packages with BouncyCastle. Besides rebasing the cert code on a supported library, these changes are pre-requisites to a future PR that will upgrade Java from 8 to 11 LTS.


## Which Traffic Control components are affected by this PR?

- Traffic Router

Documentation is not required as no features/dependencies were changed. All references to Sun.* were changed to use BouncyCastle, which is already included in ATC.

Tests are not included because at the function and class level, nothing should be different, so the existing tests should be sufficient.

## What is the best way to verify this PR?

- Run tests and confirm success
- Run CiaB and confirm that certs are loaded and used correctly when browsing HTTPS endpoints.

## The following criteria are ALL met by this PR



## Additional Information
[mailing list discussion](https://lists.apache.org/thread.html/r083c16c4feb33824e6638c43808d0866c30f63e64d31a2c0fa176001%40%3Cdev.trafficcontrol.apache.org%3E)
